static/app/components/forms/fields/numberField.tsx

```diff
@@ -80,12 +80,10 @@ function createFieldWithSuffix({suffix}: {suffix: React.ReactNode}) {
     );
   };
 }
-const SuffixWrapper = styled('span')<Omit<NumberFieldProps, 'name'>>`
-  ${p => inputStyles(p)};
+const SuffixWrapper = styled(Input.withComponent('span'))<Omit<NumberFieldProps, 'name'>>`
   color: transparent;
   background: transparent;
   border-color: transparent;
-  box-shadow: inset transparent;
```

Comment:
this `box-shadow` value was invalid, and it didn’t do anything in the browser (the browser devtools also showed it as as invalid). I double checked that the component looks the same without this setting.